<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>公文輸入輔助程式</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="Layout_inputArea" rowstretch="0,0,0,0,0,0,0,0,0" columnstretch="0,1,0,0,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_number">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>原文號　</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_number</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="lineEdit_org"/>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="lineEdit_word"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_reason">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>事由　　</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>textEdit_reason</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_word">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>原文字　</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_word</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <layout class="QHBoxLayout" name="Layout_view" stretch="3,3,0,3,0,3,0,1,3,3">
        <item>
         <widget class="QLineEdit" name="lineEdit_viewDate">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_viewWord">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_wn">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>字第</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_viewNum">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_nm">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>號</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_viewOrg">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_send">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>函</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_viewReason">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_viewName">
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_org">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>發文機關</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_org</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_date">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>日期　　</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>calendar</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="4" column="5">
       <widget class="QTextEdit" name="textEdit_reason">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_name">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>承辦人　</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="6">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_copy">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>複製這筆資料到剪貼簿</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_clear">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_clearAll">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>全部清除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_copyAll">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>複製全部資料到剪貼簿</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_viewPre">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="intValue" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_next">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_msg">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>訊息:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_nextData">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>儲存並下一筆</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="QLineEdit" name="lineEdit_number"/>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QComboBox" name="cb_word">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="cb_org">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微軟正黑體</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>勞動部勞動力發展署中彰投分署</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>臺中市政府</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>臺中市政府勞工局</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QCalendarWidget" name="calendar"/>
      </item>
      <item row="4" column="1" colspan="4">
       <layout class="QGridLayout" name="Layout_btn">
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_rmws">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>貼上
並移除空白字元</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="5" column="1" colspan="4">
       <widget class="QComboBox" name="cb_name">
        <property name="font">
         <font>
          <family>微軟正黑體</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>王淑真</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>梁瀞云</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>周恩琦</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>管采育</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>侯慧美</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>張幼麗</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>張宥熏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>許雅婷</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>calendar</tabstop>
  <tabstop>cb_org</tabstop>
  <tabstop>cb_word</tabstop>
  <tabstop>lineEdit_number</tabstop>
  <tabstop>btn_rmws</tabstop>
  <tabstop>cb_name</tabstop>
  <tabstop>btn_copy</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_clearAll</tabstop>
  <tabstop>lineEdit_viewWord</tabstop>
  <tabstop>lineEdit_viewOrg</tabstop>
  <tabstop>textEdit_viewReason</tabstop>
  <tabstop>lineEdit_viewName</tabstop>
  <tabstop>lineEdit_org</tabstop>
  <tabstop>lineEdit_word</tabstop>
  <tabstop>textEdit_reason</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_viewDate</tabstop>
  <tabstop>lineEdit_viewNum</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_org</sender>
   <signal>activated(QString)</signal>
   <receiver>lineEdit_org</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_word</sender>
   <signal>activated(QString)</signal>
   <receiver>lineEdit_word</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_name</sender>
   <signal>activated(QString)</signal>
   <receiver>lineEdit_name</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_org</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_viewOrg</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>820</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_word</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_viewWord</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_number</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_viewNum</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_viewName</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>889</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_word</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lineEdit_word</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>889</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
